### PR TITLE
connect views to db

### PR DESCRIPTION
**Pull request by fordf**

### CODE REVIEW

File: learning_journal/scripts/initializedb.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    session_factory = get_session_factory(engine)
+
+    with transaction.manager:
+        dbsession = get_tm_session(session_factory, transaction.manager)
+        for entry in ENTRIES:
+            model = Entry(title=entry['title'],
+                          body=entry['body'],
+                          creation_date=entry['creation_date'],
+                          id=entry['id'])
```
Comment (bgarnaat) [inline learning_journal/scripts/initializedb.py:69]: id should probably be handled by your ORM / DBMS as an auto-incremented integer.

File: learning_journal/views/default.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+
+# @view_config(route_name='home', renderer='../templates/home.jinja2')
+# def my_view(request):
+#     try:
+#         query = request.dbsession.query(Entry)
+#         one = query.filter(Entry.title == 'one').first()
+#     except DBAPIError:
+#         return Response(db_err_msg, content_type='text/plain', status=500)
+#     return {'one': one, 'project': 'learning_journal'}
```
Comment (bgarnaat) [inline learning_journal/views/default.py:16]: Corpse code shouldn't be in your pull requests into master.

File: learning_journal/views/default.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    latest = entries[-1]
+    left_entries, right_entries = [], []
+    for i in range(len(entries) - 1):
+        if i % 2:
+            left_entries.append(entries[i])
+        else:
+            right_entries.append(entries[i])
+    return {'latest': latest,
+            'left_entries': left_entries,
+            'right_entries': right_entries}
```
Comment (bgarnaat) [inline learning_journal/views/default.py:32]: Left / right could be handled by your template.  You can then return just a single list of entries to your template and apply  a left/right class as you iterate through building your index page.

Comment (fordf) [reply learning_journal/views/default.py:32]: But it's easier in python :(

**Pull request merged**